src/Delegate.Daxif/RELEASE_NOTES.md

```diff
@@ -1,4 +1,7 @@
 # Release Notes
+### 5.5.0 - February 06 2023
+* Added 'useUniqueInstance' parameter to GetCrmServiceClient() with default value: 'true'. - This enables a single script to correctly connect to multiple environments, while still having the option to set the parameter to 'false' in order to optimize any existing scripts that may be adversely affected regarding performance by this change.
```

Comment:
This seems to contradict the implementation, where the default value is `false` for backwards compatibility and the parameter must be explicitly set to `true`.

Reply:
Fixed to default to 'true' in code.